Formula/filebeat.rb

```diff
@@ -1,19 +1,11 @@
 class Filebeat < Formula
-  desc "File harvester, used to fetch log files and feed them into logstash"
+  desc "File harvester to ship log files to Elasticsearch or Logstash"
   homepage "https://www.elastic.co/products/beats/filebeat"
-  url "https://github.com/elastic/beats/archive/v1.2.3.tar.gz"
-  sha256 "8eea85de415898c362144ba533062651d8891241c738799e54cc9b17040c1fc9"
+  url "https://github.com/elastic/beats/archive/v5.0.0.tar.gz"
+  sha256 "3e6b7cf2ee5f52e78ae87ef04ab9dd49977c89f86a09416586896aeaea844e34"
 
   head "https://github.com/elastic/beats.git"
 
-  bottle do
```

Comment:
Ok, thought it had to be removed every time as it is auto generated. Still new to homebrew contributions. Put it back in.